fixtures/dummy/app/components/comment.html.erb

```diff
@@ -0,0 +1,15 @@
+<div>
+  <span>
+    <%= @name %>
+  </span>
+  <span>
+    <%= @body %>
+  </span>
+
+  <%= content %>
+
+  <%= render Views::Card.new do |a| %>
+    <p>Start Card A</p>
+    <%= a.title "Hello from A" %>
+  <% end %>
```

Comment:
I like this, but can we put the `@name` and `@body` instance variables in the card to be sure they work in the block?